app/Http/Controllers/ReferencesController.php

```diff
@@ -241,28 +241,27 @@ public function update(Request $request, ?Reference $reference = null)
     /**
      * Delete the particular reference from storage.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \App\Models\Reference  $reference
+     * @param  \Illuminate\Http\Request $request
+     * @param  \App\Models\Reference    $reference
      * @return \Illuminate\Http\Response
      */
     public function destroy(Request $request, Reference $reference)
     {
         $this->authorize('delete', $reference);
 
         //TODO: when projects exist independently on profile, delete seperatley
-        foreach($reference->projects as $project) {
+        foreach ($reference->projects as $project) {
```

Comment:
Any chance we could utilize mass deleting here, instead of firing off individual delete calls in a loop?

From the Laravel docs:

>You can also run a delete statement on a set of models. In this example, we will delete all flights that are marked as inactive. Like mass updates, mass deletes will not fire any model events for the models that are deleted
```php
$deletedRows = App\Flight::where('active', 0)->delete();
```

Reply:
Yep, it can be done here with `$reference->projects()->delete();`